src/cmd/wallet.rs

```diff
@@ -579,7 +636,7 @@ pub fn run_split_cmd(wallet: &GunWallet, opt: SplitOpt) -> anyhow::Result<CmdOut
                 // add one recipient so we at least get one split utxo of the correct size.
                 .add_recipient(
                     bdk_wallet
-                        .get_change_address(AddressIndex::New)?
+                        .get_change_address(AddressIndex::LastUnused)?
```

Comment:
Hmm so this bit doesn't fix the underlying problem with `split_change`. Perhaps for now the best thing to do is to make the number of outputs mandatory so we don't have to use `split_change`. 

After this is merged we should make a PR to BDK adding `FirstUnused` and batch address fetch to BDK and maybe introduce `split_change` with that.